Library/Formula/emp.rb

```diff
@@ -0,0 +1,27 @@
+require "language/go"
+
+class Emp < Formula
+  desc "CLI for Empire."
+  homepage "https://github.com/remind101/emp"
+  url "https://github.com/remind101/emp/archive/v0.9.1.tar.gz"
+  version "0.9.1"
+  sha256 "9619ffd25856988d92d39b31d9e7ed6edacbd85543ff6e60300954df7c16135d"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+
+    emppath = buildpath/"src/github.com/remind101/emp"
+    emppath.install Dir["{*,.git}"]
+
+    cd emppath do
+      system "go", "build", "-o", bin/"emp"
+    end
+  end
+
+  test do
+    assert_equal "#{version}",
```

Comment:
I know that according to the [testing guidelines](https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Formula-Cookbook.md#add-a-test-to-the-formula) one should try and avoid testing the formula with `--version` however this CLI typically operates against an HTTP server. 

If it's preferred, I could write a mock http server that responds to the CLI's requests and test a "simple" route such as listing apps.